server-spi/src/main/java/org/keycloak/models/UserSessionProvider.java

```diff
@@ -138,6 +138,11 @@ UserSessionModel createUserSession(String id, RealmModel realm, UserModel user,
     void removeUserSession(RealmModel realm, UserSessionModel session);
     void removeUserSessions(RealmModel realm, UserModel user);
 
+    /**
+     * Remove expired events in all the realms.
```

Comment:
Should be here rather something like "Remove expired user sessions in all the realms" ?

Reply:
Done. I've added "Remove expired user sessions and client sessions in all the realms" as it applies to client sessions as well.